templates/networkPolicy.go

```diff
@@ -0,0 +1,72 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	openshiftv1 "github.com/openshift/api/network/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var IngressNetworkPolicyTemplate = netv1.NetworkPolicy{
+	Spec: netv1.NetworkPolicySpec{
+		Ingress: []netv1.NetworkPolicyIngressRule{
+			{
+				From: []netv1.NetworkPolicyPeer{
+					{
+						PodSelector: &metav1.LabelSelector{},
+					},
+				},
+			},
+		},
+		PolicyTypes: []netv1.PolicyType{
+			netv1.PolicyTypeIngress,
+		},
+		PodSelector: metav1.LabelSelector{},
+	},
+}
+
+var EgressNetworkPolicyTemplate = openshiftv1.EgressNetworkPolicy{
+	Spec: openshiftv1.EgressNetworkPolicySpec{
+		Egress: []openshiftv1.EgressNetworkPolicyRule{
+			{
+				To: openshiftv1.EgressNetworkPolicyPeer{
+					DNSName: "",
+				},
+				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
+			},
+			{
+				To: openshiftv1.EgressNetworkPolicyPeer{
+					DNSName: "",
+				},
+				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
+			},
+			{
+				To: openshiftv1.EgressNetworkPolicyPeer{
+					DNSName: "events.pagerduty.com",
+				},
+				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
+			},
+			{
+				To: openshiftv1.EgressNetworkPolicyPeer{
+					CIDRSelector: "0.0.0.0/0",
+				},
+				Type: openshiftv1.EgressNetworkPolicyRuleDeny,
+			},
+		},
+	},
+}
```

Comment:
Let's keep a single resource template in a file, so we will have
`templates/networkpolicy.go` and `templates/egressnetworkpolicy.go` (matching the names of the resource kind)

While doing the change can you please also rename all other template files to lower case, the different operating system has different file systems that might or might not be case sensitive.
The offending files are: 
`dmsPrometheusRule.go` -> `dmsprometheusrule.go`
`k8sMetricsServiceMonitor.go` -> `k8smetricsservicemonitor.go`